### PR TITLE
PMM-7 Simplify CI

### PR DESCRIPTION
**Pull request by artemgavrilov**
PMM-7

### CODE REVIEW

**Comment by artemgavrilov** [comment]
@BupycHuk it requires required job renaming `merge-gatekeeper` -> `Merge Gatekeeper`

**Review by BupycHuk — APPROVED** [review]

**Comment by BupycHuk** [comment]
@artemgavrilov ping me when it's ready to merge, I'll merge and update settings.

File: .github/workflows/admin.yml [hunk 1]
```diff
@@ -33,20 +33,14 @@ jobs:
...
     defaults:
       run:
-        working-directory: ${{ env.WORKING_DIR }}/admin
+        working-directory: ${{ github.workspace }}/admin
 
     steps:
-      - name: Check out code into the working directory under the GOPATH
+      - name: Check out code
         uses: percona-platform/checkout@v3
-        with:
```
Comment (BupycHuk) [inline .github/workflows/admin.yml:47]: what is the reason to remove it? I think it speeds up CI work.

Comment (artemgavrilov) [reply .github/workflows/admin.yml:47]: Because it's default value: https://github.com/actions/checkout#usage

**Review by denisok — APPROVED** [review]

**Comment by artemgavrilov** [comment]
@BupycHuk We can move with this

**Pull request merged**